### DISCUSSION TITLE
SwaggerUI integration.

### DISCUSSION THREAD

**Discussion by marcem7D0** [loco-rs/loco#334] [Q&A] [answered]
Hello!

Is it possible to auto generate swagger-ui for rest apis with Loco, like you can with Poem OpenApi for example?

**Comment by jondot** [comment, marked as answer]
Did you explore solutions and libraries that are mentioned here? https://github.com/tokio-rs/axum/issues/50

**Reply by marcem7D0** [reply]
No, I did not, but thanks for pointing out, will check!

**Comment by wvffle** [comment]
@mmittnacht did some library work out for you? I've tried aide and utoipa and I couldn't get them to work nicely.
